TempleEight/docs/guide/temple-test.md

```diff
@@ -3,3 +3,61 @@ id: temple-test
 title: Temple Test
 sidebar_label: Temple Test
 ---
+
+In addition to automatic generation of microservice-based infrastructure, Temple also offers a tool for testing the generated code.
+
+`temple test` has two main operating modes:
+
+1. A full testing mode, where it will spin up the infrastructure locally, mock a request to each endpoint and then shut the infrastructure down
```

Comment:
```suggestion
1. A full testing mode, where it will spin up the infrastructure locally, mock a request to each endpoint and then shut the infrastructure down.
```